pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -1551,17 +1315,6 @@ func (cc *XController) Run(stopCh chan struct{}) {
 
 	go cc.queueJobInformer.Informer().Run(stopCh)
 
-	go cc.qjobResControls[arbv1.ResourceTypePod].Run(stopCh)
```

Comment:
pod controller is needed and is being actively used

Reply:
Updated with a subset of the pod controller reinstated, i.e., implementing only the `UpdateQueueJobStatus` method.